docs/getting-started/summaryofchanges.md

```diff
@@ -43,15 +49,75 @@ The following features and enhancements were added.
-- Add `encoding` option to `zosmf` profile type.
-- Added API to delete migrated data sets.
+- Added the ability to log into and out of API ML using a token. [#718](https://github.com/zowe/zowe-cli/issues/718)
+
+- Added the `--base-profile` option to all commands that use profiles to let them make use of base profiles that contain shared values. [#718](https://github.com/zowe/zowe-cli/issues/718)
+
+- CLI commands now prompt for any of the following option values if the option is missing: host, port, user, and password. [#718](https://github.com/zowe/zowe-cli/issues/718)
+
+- Added character encoding/code page support for download and upload data set operations in the API library and the CLI. [#632](https://github.com/zowe/zowe-cli/issues/632)
+
+- Added the `--encoding` option to the `zosmf` profile type. [#632](https://github.com/zowe/zowe-cli/issues/632)
+
+- Introduced an API to delete migrated data sets. [#715](https://github.com/zowe/zowe-cli/issues/715).
 
 The following features and enhancements were added to the **Imperative CLI Framework**:
 
+- Added the `ConnectionPropsForSessCfg.addPropsOrPrompt` function to store credentials, such as a token, in a session configuration object. [#718](https://github.com/zowe/zowe-cli/issues/718)
+
+    - CLI plug-ins must implement this function to create sessions in order to consume automatic token-handling and prompt for mission options features.
+
+    - Connection information is obtained from the command line in the following order: Environment variables, service profiles, base profiles, or a default option value.
+
+    - If connection information is not supplied to any core CLI command, the user is prompted for:
+        -  host
+        -  port
+        -  user
+        -  password
+
+      The prompt times-out after 30 seconds so that automated scripts will not fail.
+
+- Added base profiles, a type of profile that can store values and provide them to other profiles types, such as zosmf profiles. [#402](https://github.com/zowe/imperative/pull/402)
+
+  The following properties can be stored in a base profile:
+
+  - host
+  - port
+  - user
+  - password
+  - rejectUnauthorized
+  - tokenType
+  - tokenValue
+
+  To implement base profiles in a CLI plug-in, define a `baseProfile` schema on your Imperative configuration object. If the `baseProfile` schema is defined, base profile support is added to any command that uses profiles.
+
+- Added `login` and `logout` commands to retrieve and delete tokens. [#405](https://github.com/zowe/imperative/issues/405)
+
+  - Added a `showToken` flag to display the token and not save it to the user profile.
+
+  - Added the ability to create a user profile upon login, if no profile of that type existed previously.
+
+- Sensitive session properties, such as user, password, and token value, are now hidden in log files. Prior to this release, only the password was hidden. [#416](https://github.com/zowe/imperative/issues/416)
```

Comment:
This point doesn't need to be included. It was a regression that happened after Zowe 1.12 and was fixed before Zowe 1.13. Only users consuming Imperative from Zowe Artifactory/public NPM were affected.